tests/cases/fourslash/smartIndentObjectBindingPattern01.ts

```diff
@@ -1,14 +1,16 @@
 ﻿/// <reference path="fourslash.ts"/>
 
 ////var /*1*/{/*2*/a,/*3*/b:/*4*/k,/*5*/
- 
+
 function verifyIndentationAfterNewLine(marker: string, indentation: number): void {
     goTo.marker(marker);
     edit.insert("\r\n");
     verify.indentationIs(indentation);
 }
 
-verifyIndentationAfterNewLine("1", 4);
+// TODO (arozga): fix this.

```

Comment:
No. This problem actually already existed, but only by adjusting the caret correctly is the issue exposed. But actually fixing indenting behaviors is outside the scope of this PR. When this change get's accepted, I'll make a new issue for this.